emission/net/api/cfc_webapp.py

```diff
@@ -482,6 +483,25 @@ def getUUID(request, inHeader=False):
         traceback.print_exc()
         abort(403, e)
 
+def resolve_auth(auth_method):
```

Comment:
Seems like you can add a unit test for `auth_method` that uses one of the study configs that always exist - e.g. devapp/stage.